include/seqan3/alphabet/nucleotide/dna4.hpp

```diff
@@ -132,6 +134,32 @@ struct dna4
     //!\brief The size of the alphabet, i.e. the number of different values it can take.
     static constexpr rank_type value_size{4};
 
+    /*!\name Conversion operators
+     * \{
+     */
+    //!\brief Implicit conversion between dna* and rna* of the same size.
```

Comment:
Then I think the documentation should directly refer to dna4 and rna4. (and not `dna*`)